agent/windows-setup-agent/SetupWizard.cs

```diff
@@ -194,36 +194,36 @@ private void ConfigureService()
 				string master_host, master_port;
 				GetMasterHostPort(out master_host, out master_port);
 
-				args += " --master_host " + Convert.ToString(master_host).Trim()
-				    + "," + Convert.ToString(master_port).Trim();
+				args += " --master_host " + master_host.Trim()
+				    + "," + master_port.Trim();
```

Comment:
I just noticed that you're calling Trim() on the strings that were returned by GetMasterHostPort(). That's not strictly necessary, because you're already calling Trim() in that function.